emails/templates/emails/wrapped_email.html

```diff
@@ -1,3 +1,7 @@
+{% load ftl %}
+{% load email_extras %}
+{% withftl mode='server' bundle='privaterelay.ftl_bundles.main' language=recipient_profile.language %}
```

Comment:
```suggestion
{% withftl mode='server' bundle='privaterelay.ftl_bundles.main' language=recipient_language %}
```

Reply:
I like putting "fat" objects into templates rather than fine-grained values. There's minimal performance impact (unless the object is VERY large), and it makes future code easier. So when future Luke opens this template again, he will see that `recipient_profile` is available, and can use any other property on it.